pallets/connectors/src/message.rs

```diff
@@ -233,6 +517,13 @@ mod tests {
 
 	pub type ConnectorMessage = Message<Domain, PoolId, TrancheId, Balance, Rate>;
 
+	const AMOUNT: Balance = 100000000000000000000000000;
+	const POOL_ID: PoolId = 12378532;
+	const TOKEN_ID: u128 = 246803579;
+	const TRANCHE_HEX: &str = "811acd5b3f17c06841c7e41e9e04cb1b";
```

Comment:
I'd rather have `TRANCHE_ID: TrancheId = tranche_id_from_hex("811acd5b3f17c06841c7e41e9e04cb1b");` so that we don't have to call `tranche_id_from_hex` every time

Reply:
That was my initial idea. Unfortunately that does not work as  calls in constants are limited to constant functions, tuple structs and tuple variants. Since `from_hex` is not a constant function, I feel like our hands are tied.

Another option could be to have default `get` functions instead of constants. WDYT? E.g.

```
	fn default_address20(hex: &str) -> [u8; 20] {
		<[u8; 20]>::from_hex("1231231231231231231231231231231231231231").expect("Should be valid 20 bytes")
	}
```

Reply:
@wischli that's why I din't include the `const` bit in my suggestion above; but your suggestion is also good :)